### PR TITLE
begin categories class with add fucntion

### PR DESCRIPTION
**Pull request by ArtemNikolaev**

### CODE REVIEW

**Comment by pavelgrinevich** [comment]
image

File: src/components/widgets/ModalCategories.svelte [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+  import store from '../../utils/store';
+  import { close, name } from '../../models/modalManager';
+  import { Category } from '../../models/categories';
 
-	function mapCategories(name) {
-	    return { name, old: name, disabled: true};
-	}
-	let categories = store.getState().categories.map(mapCategories);
-	store.subscribe(() => categories = store.getState().categories.map(mapCategories));
+  let newCategory = new Category();
```
Comment (pavelgrinevich) [inline src/components/widgets/ModalCategories.svelte]: I prefer `const` in this case
`const newCategory = new Category();`

**Comment by krivonos28** [comment]
I can!

File: src/components/widgets/ModalCategories.svelte (hunk 1 above)
Comment (ArtemNikolaev) [reply src/components/widgets/ModalCategories.svelte]: Theres some problem with reactivity in svelte, view don't see changes in newCategory. so i need to do stuff like `newCategory = newCategory`. And this stuff will not work with `const`

Sad but true

**Pull request merged**